src/main/java/com/frameworkium/core/ui/ExtraExpectedConditions.java

```diff
@@ -54,16 +87,20 @@
      * @see ExtraExpectedConditions#notPresentOrInvisible(WebElement)
      */
     public static ExpectedCondition<List<? extends WebElement>> notPresentOrInvisible(
-            final List<? extends WebElement> elements) {
+            final List<? extends WebElement> elements, int atLeast) {
```

Comment:
Others might be using the original signature in their tests, we shouldn't remove things from the public API without deprecating first, this could be easily fixed by creating a new method with the old signature but just passed in 0 as the default.